test/util_test.js

```diff
@@ -204,9 +230,14 @@ describe('createDataset()', function() {
       createDataset: create,
     });
 
+    const mockSettings = {
```

Comment:
Nit: can we use consistent naming here? (i.e. `settingsMock`)

If you choose to change this, apply this comment (`mockSettings` --> `settingsMock`) throughout your PR.